### DISCUSSION TITLE
I cannot claim land???

### DISCUSSION THREAD

**Discussion by xdjoee** [GriefPrevention/GriefPrevention#1868] [Q&A]
![image](https://user-images.githubusercontent.com/69331937/165867558-a71be19d-a598-4c54-b253-a8c01af8f9f1.png)
It doesnt allow me to claim land

**Comment by Jikoo** [comment]
You appear to have entered subdivision creation mode. Unequip and re-equip the shovel and don't run `/subdivideclaims`.
